### ISSUE TITLE
Install-RequiredModule error on Mac/Linux

### ISSUE THREAD

**Issue by bravo-kernel** [PoshCode/ModuleBuilder#92]
Unsure if this is related to the module update but Azure Pipelines (for Mac and Linux) is choking on Install-RequiredModule with the following error:

    Unable to find type [VersionRange]

Looks to be caused by this pipeline template: https://github.com/PoshCode/Azure-Pipelines/blob/master/Install-RequiredModule-step.yml#L34 (already mentioning Windows only).

Pipeline details found here:  
https://dev.azure.com/alt3bv/Docusaurus.Powershell/_build/results?buildId=620&view=logs&j=18fe8392-28c7-5583-f88f-5fc6a2f5d637&t=8a5ae15e-4367-56d2-ffbc-efca611e9efb

No issues on Windows:

![image](https://user-images.githubusercontent.com/230500/76688453-40815280-662d-11ea-8f4a-a536ed7d720e.png)

Pointers appreciated

**Comment by Jaykul** [comment]
I don't _think_ that's related. The `[VersionRange]` type is from the `NuGet.Versioning` assembly, which should be embedded in `Install-RequiredModule` (see [RequiredModules](https://github.com/Jaykul/RequiredModules/tree/master/source/lib))

**Comment by bravo-kernel** [comment]
Thanks for the feedback. I actually looked at that repo but it's where I got really confused:

- Install-RequiredModule vs. Install-RequiredModules (with trailing s)
- Both are on PSGallery
- the first one is referred to in the pipeline azure yaml
- the second one, I could not find any references to 

Since the code is in different repos it's a bit hard to add debugging. Let's leave this open for now, perhaps a light bulb moment will happen.
